src/main/kotlin/com/exactpro/th2/read/file/common/AbstractFileReader.kt

```diff
@@ -771,6 +795,11 @@ abstract class AbstractFileReader<T : AutoCloseable>(
     companion object {
         private val LOGGER = KotlinLogging.logger { }
 
+        const val MESSAGE_STATUS_PROPERTY = "STATUS"
```

Comment:
I think we should keep properties names more unique to avoid possible collisions
```suggestion
        const val MESSAGE_STATUS_PROPERTY = "th2.read.order_marker"
```